build_bundle_image.sh

```diff
@@ -267,29 +267,36 @@ function prepare_temp_directory {
 
 	download "${tomcat_download_dir}/apache-tomcat.zip" "${tomcat_url}"
 
-	mv "${TEMP_DIR}/liferay/tomcat" "${TEMP_DIR}/liferay/tomcat-temp"
+	local tomcat_folder="tomcat"
```

Comment:
@kiwm, can you please rename this variable to `tomcat_dir_name`?

This is the pattern we have in place in the scripts to name folder names. Please look for it. :slightly_smiling_face: